src/transaction/log_applier.c

```diff
@@ -652,7 +668,15 @@ la_log_phypageid (LOG_PAGEID logical_pageid)
 static int
 la_log_io_read (char *vname, int vdes, void *io_pgptr, LOG_PHY_PAGEID pageid, int pagesize)
 {
-  return la_log_io_read_with_max_retries (vname, vdes, io_pgptr, pageid, pagesize, -1);
+  int err = NO_ERROR;
+  LOG_PAGE *log_pgptr = (LOG_PAGE *) io_pgptr;
+
+  err = la_log_io_read_with_max_retries (vname, vdes, io_pgptr, pageid, pagesize, -1);
+  if (err != NO_ERROR)
+    {
+      return err;
+    }
```

Comment:
I think this if clause can be skipped

Reply:
definitely.... Thanks, I missed that code which I should've reverted.

Reply:
commit: [bdcc1a8](https://github.com/CUBRID/cubrid/pull/2450/commits/bdcc1a8a7e993a6f67f991ce420ed06914524cd8)